jenkins/release-workflows/release-chores.jenkinsfile

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+lib = library(identifier: 'jenkins@8.2.0', retriever: modernSCM([
+    $class: 'GitSCMSource',
+    remote: 'https://github.com/opensearch-project/opensearch-build-libraries.git',
+]))
+
+pipeline {
+    options {
+            timeout(time: 1, unit: 'HOURS')
+    }
+    agent {
+        docker {
+            label 'Jenkins-Agent-AL2023-X64-M54xlarge-Docker-Host'
+            image 'opensearchstaging/ci-runner:ci-runner-centos7-opensearch-build-v3'
```

Comment:
This is severely outdated can we use the rockylinux8 one?

Reply:
Sure can you let me know the exact image?

Reply:
[ci-runner-rockyolinux8-opensearch-build-v1](https://hub.docker.com/layers/opensearchstaging/ci-runner/ci-runner-rockyolinux8-opensearch-build-v1/images/sha256-d16f968df2ae9487c3cf6593999523d889653b7def043fc028aba7025db1bb22)

Reply:
Please test this image beforehand now sure if it has all the deps you need.

Reply:
This image does not have recent version of `curl` for awssig4 option. 
```
curl: option --aws-sigv4: is unknown
curl: try 'curl --help' or 'curl --manual' for more information
```
Is this image fine? https://github.com/opensearch-project/opensearch-build/blob/main/jenkins/integ-test-notification.jenkinsfile#L20
Metrics cluster needs sigv4 option.

Reply:
Yeah AL2 is fine for the moment.

Reply:
Tested with above image. Works as expected
https://github.com/gaiksaya/opensearch-build/issues/49#issuecomment-2664147304
Thanks!